### ISSUE TITLE
[NEW] Let the user reply messages directly from Notification

### ISSUE THREAD

**Issue by rafaelks** [RocketChat/Rocket.Chat.Android#530]
# Description

We need to let the user reply to a message from the Notification screen. Here's a new example from how WhatsApp is doing it:

![img_0370](https://user-images.githubusercontent.com/551004/31589080-f8d763ae-b1af-11e7-91ed-1c9bfff2a5dd.jpg)

**Comment by amritsinghcse** [comment]
I would like to work on this feature

**Comment by leonardoaramaki** [comment]
@SuperHaker Thanks for your interest on contributing!
At this very moment though, we're on a refactoring move of the push notification handler.
I'll soon open a PR of the ongoing progress so that you and others could follow. 
So it's better if you follow this one before starting working on this.

**Comment by amritsinghcse** [comment]
Okay. I'll keep an eye

**Comment by leonardoaramaki** [comment]
@SuperHaker this is the PR (https://github.com/RocketChat/Rocket.Chat.Android/pull/541) you should work upon if you want to follow it. Feel free to send another PR or any comments.

**Comment by leonardoaramaki** [comment]
Already implemented on the new app.
